### ISSUE TITLE
tfx-cli@0.6.0 regression: `extension create` fails without internet access

### ISSUE THREAD

**Issue by nil4** [Microsoft/tfs-cli#285]
We use an on-premise TFS 2018 Update 2 instance (version 16.131.27701.1), hosted on a network with no Internet access, and build TFS extensions locally. We package our custom TFS extensions using:

```cmd
node_modules\.bin\tfx.cmd extension create --root tasks
```
This worked well, and produced a VSIX, with every version before `tfx-cli 0.6.0`. When upgrading to v0.6.0 today, however, the same command now fails:

```
[37m[90mTFS Cross Platform Command Line Interface v0.6.0[37m[39m
[37m[90mCopyright Microsoft Corporation[37m[39m
##[error][41merror: Error: getaddrinfo EAI_AGAIN app.vssps.visualstudio.com app.vssps.visualstudio.com:443[49m
##[error]Cmd.exe exited with code '-1'.
```

It does not look expected that running `tfx-cli extension create` would attempt to connect to visualstudio.com. This seems related to changes committed in https://github.com/Microsoft/tfs-cli/commit/5bfdc01bcd8ff52e5ecbd58e3cdc623d7d0ea343.

Will an Internet connection be required for packaging extensions? We would be blocked from upgrading past v0.5.14 if that is the case.

**Comment by nil4** [comment]
The output of the same TFS build, running the same command line, but using `tfx-cli` v0.5.14 is:

```
[37m[90mTFS Cross Platform Command Line Interface v0.5.14[37m[39m
[37m[90mCopyright Microsoft Corporation[37m[39m
[37m[32m[37m[39m
[37m[32m=== Completed operation: create extension ===[37m[39m
[37m - VSIX: F:\1\113\s\REDACTED-REDACTED.redacted-deployment-1.0.241.vsix[39m
[37m - Extension ID: redacted-deployment[39m
[37m - Extension Version: 1.0.241[39m
[37m - Publisher: REDACTED-REDACTED[39m
```

**Comment by nil4** [comment]
Updating the v0.6.0 build command to:

```shell
node_modules\.bin\tfx.cmd extension create --root tasks --trace-level debug --debug-log-stream stdout
```

Gives a hint about the source of the issue `GetAddrInfoReqWrap.onlookup [as oncomplete] (dns.js:57:26)`:

```
[90m[36m2018-09-02T15:37:56.968Z : [90mloader.load[39m
[37m[90mTFS Cross Platform Command Line Interface v0.6.0[37m[39m
[37m[90mCopyright Microsoft Corporation[37m[39m
[90m[36m2018-09-02T15:38:10.704Z : [90mError: getaddrinfo EAI_AGAIN app.vssps.visualstudio.com app.vssps.visualstudio.com:443[39m
[90m[36m2018-09-02T15:38:10.704Z : [90m    at GetAddrInfoReqWrap.onlookup [as oncomplete] (dns.js:57:26)[39m
[41merror: Error: getaddrinfo EAI_AGAIN app.vssps.visualstudio.com app.vssps.visualstudio.com:443[49m
Cmd.exe exited with code '-1'.
```

**Comment by trevorsg** [comment]
Fixed in 0.6.1. Thanks for reporting!
